hub/ui/src/components/search-bar/SearchBar.tsx

```diff
@@ -36,8 +36,11 @@ const SearchBar: React.FC = (props: any) => {
   const onTextChanged = (e: any) => {
     const value = e;
     let suggestions: any = [];
-    const regex = new RegExp(`${value}`, 'i');
-    suggestions = props.TaskDataList.sort().filter((v: any) => regex.test(v.name));
+    const nameregex = /-/gi;
+    const regex = new RegExp(`${ value.replace(nameregex, ' ') }`, 'i');
```

Comment:
have you considered using some library like fuzzysearch? https://github.com/bevacqua/fuzzysearch ? instead of using regex?

Reply:
have replaced regex with fuzzysearch